Views/ViewModifiers/ExternalLinkHandler.swift

```diff
@@ -11,12 +11,11 @@ import SwiftUI
 import Defaults
 
 struct ExternalLinkHandler: ViewModifier {
+    @Binding var externalURL: URL?
```

Comment:
```suggestion
    @Binding var externalLinkURL: URL?
```
(following the 2nd feedback from above)